docs/src/replication.md

```diff
@@ -59,9 +59,6 @@ CREATE USER streaming_replica WITH REPLICATION;
    -- NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB NOBYPASSRLS
 ```
 
-!!! Note
```

Comment:
@wadlejitendra why don't we then remove the superuser code?

Reply:
Given that the scope of this ticket is to remove support from version 1.19, please make sure that we remove the setting of superuser in streaming_replica which was required in Postgres 10.